plugin/pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -377,6 +378,17 @@ func TestGenericScheduler(t *testing.T) {
 			expectsErr: true,
 			wErr:       fmt.Errorf("persistentvolumeclaim \"existingPVC\" is being deleted"),
 		},
+		{
+			// alwaysCheckAllPredicates is true
```

Comment:
Sorry, but this is not useful. You should probably add a couple of predicates and create a pod that fails all predicates and then check that all of the predicates are exercised when `alwaysCheckAllPredicates` is true.

Reply:
@bsalamat  Thank you very much for your guidance, I have already done it.